users/forms.py

```diff
@@ -0,0 +1,29 @@
+from django.contrib.auth.forms import UserCreationForm, UserChangeForm
+
+from users.models import CustomUser
+
+class CustomUserCreationForm(UserCreationForm):
+    """
+    A form that creates a user, with no privileges, from the given email and
+    password.
+    """
+
+    def __init__(self, *args, **kargs):
+        super(CustomUserCreationForm, self).__init__(*args, **kargs)
+
+    class Meta:
+        model = CustomUser
+        fields = ("email",)
+
+class CustomUserChangeForm(UserChangeForm):
+    """A form for updating users. Includes all the fields on
+    the user, but replaces the password field with admin's
+    password hash display field.
+    """
+
+    def __init__(self, *args, **kargs):
```

Comment:
Same as above

Reply:
Same as before, as in I don't think this __init__ is necessary as well.

Reply:
Same here

Reply:
Done